DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.py

```diff
@@ -59,10 +57,16 @@ def filterReplicas( opFile, logger = None, dataManager = None, seCache = None ):
 
   replicas = replicas["Successful"].get( opFile.LFN, {} )
 
+  # FC checksum is more likely better than opFile.Checksum
+  fcMetadata = FileCatalog().getFileMetadata( opFile.LFN )
```

Comment:
In almost all cases it is not set... I think it was there just in case
Transformations don't set it, this is why we have this problem at RAL